.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         mcVersion: ['1.8.8', '1.9.4', '1.10.2', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16.5']
         include:
          - javaVersion: 16
-           mcVersion: '1.17.1'
+           mcVersion: ['1.17', '1.17.1']
```

Comment:
no that doesn't work,
do 
 - javaVersion: 16
           mcVersion: ['1.17']

Reply:
then how would we add 1.17.1 tests?

Reply:
that's how, you put both, like this:
```
         - javaVersion: 16
           mcVersion: '1.17'
         - javaVersion: 16
           mcVersion: '1.17.1'
```

Reply:
I'll do this, but is there a cleaner way to do this for the future?

Reply:
maybe but I don't know it